### PR TITLE
ui related update

### PR DESCRIPTION
**Pull request by mingfanghu123**

### CODE REVIEW

**Review by SiyuanChen-main — APPROVED** [review]

File: src/ui.c [hunk 1]
```diff
@@ -1,56 +1,104 @@
...
 }
 
-void get_pass(char* password)
+void ui_cls()
 {
-	return;
+	system(CLS_COMMAND);
 }
 
-void route_admin(p_user_t* user_list, p_stu_t* stu_list, char* str1, char* str2)
```
Comment (deveshhatkar) [inline src/ui.c:23]: I use this method in my part

File: src/ui.c [hunk 2]
```diff
@@ -1,56 +1,104 @@
...
+void ui_pause()
 {
-	return;
+	gets(ui_input_buffer);
 }
 
-void route_guest(p_stu_t stu_list)
```
Comment (deveshhatkar) [inline src/ui.c:28]: I use this method in my part

**Review by deveshhatkar — COMMENTED** [review]
The methods I use have been changed!

**Comment by SiyuanChen-main** [comment]
> The methods I use have been changed!

As far as I know, these methods are temporaily simplified to make release 1 run. Please create a version only for release 1 and then simplify your code (remove some functions for release 2). A new pull request should be created for your part.

**Comment by SiyuanChen-main** [comment]
> > The methods I use have been changed!
> 
> As far as I know, these methods are temporaily simplified to make release 1 run. Please create a version only for release 1 and then simplify your code (remove some functions for release 2). A new pull request should be created for your part.

We need to at least complete a version (release 1) that can run, instead of uploading code that are still in development progress

**Review by LoreleiCao — APPROVED** [review]

**Review by deveshhatkar — APPROVED** [review]

**Pull request merged**